rmw_fastrtps_shared_cpp/include/rmw_fastrtps_shared_cpp/qos.hpp

```diff
@@ -23,8 +23,8 @@ namespace eprosima
 {
 namespace fastrtps
 {
-class SubscriberAttributes;
 class PublisherAttributes;
+class SubscriberAttributes;
```

Comment:
This is pre-existent, but there is no reason to forward-declare here.
The corresponding headers should be included.

Reply:
Oops. I had an alphabetical sort done on these. I'll remove this change. 
Maybe removing the forward-declaration all together can be in a different PR?

Reply:
Yes, it can be done in a different PR.